azure-pipelines.yml

```diff
@@ -43,21 +43,16 @@ stages:
         steps:
           - checkout: self
             lfs: true
-          - task: UseDotNet@2
-            displayName: "Install .NET Core 3.1 SDK"
-            inputs:
-              packageType: "sdk"
-              version: 3.1.x
           - task: UseDotNet@2
             displayName: "Install .NET Core 6.0 SDK"
             inputs:
               packageType: "sdk"
               version: 6.0.x
           - task: UseDotNet@2
-            displayName: "Install .NET Core SDK"
+            displayName: "Install .NET Core 8.0 SDK"
             inputs:
               packageType: "sdk"
-              useGlobalJson: true
```

Comment:
What made you want to switch from the global.json being used to specifying the .NET 8 SDK manually? Just consistency with the others?

Reply:
Ideally, you would use latest available patch when building project as opposed to whatever is laying around in your global.json, unless there is a compatibility issue. This was the reason of removal